specification/compute/resource-manager/Microsoft.Compute/RecommenderRP/ComputeDiagnosticBase.tsp

```diff
@@ -0,0 +1,61 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.Compute;
+/**
+ * Contains metadata of a diagnostic type
+ */
+@singleton("spot")
+@subscriptionResource
+@parentResource(SubscriptionLocationResource)
+model ComputeDiagnosticBase
+  is Azure.ResourceManager.ProxyResource<DiagnosticProperties> {
+  ...ResourceNameParameter<
+    Resource = ComputeDiagnosticBase,
+    KeyName = "placementScore",
+    SegmentName = "placementScores",
+    NamePattern = ""
+  >;
+}
+
+@armResourceOperations
+interface ComputeDiagnosticBases {
+  /**
+   * Gets Spot Placement Scores metadata.
+   */
+  #suppress "@azure-tools/typespec-azure-core/no-openapi" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @operationId("SpotPlacementScores_Get")
```

Comment:
If you're not trying to keep the operation id for backward compatibility with any existing rest api docs, then you dont need this line or the suppression for it.

Reply:
Readded suppression as it's needed to pass build errors